README.md

```diff
@@ -1,6 +1,8 @@
 Iris
 ====
 
+[![Join the chat at https://gitter.im/SciTools/iris](https://badges.gitter.im/SciTools/iris.svg)](https://gitter.im/SciTools/iris?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
```

Comment:
Alternatively we merge this and then add a follow-up PR to remove the blank line...